app/views/documents/_actions.html.erb

```diff
@@ -14,7 +14,8 @@
         <%= presenter.publish_text %>
 
         <% if presenter.publish_button_visible? %>
-          <button name="submit" class="btn btn-danger" data-module="confirm" data-message="<%= presenter.publish_alert %>" data-disable-with="Publishing..." >Publish</button>
+          <button name="submit" class="btn btn-danger" data-module="confirm" data-message="<%= presenter.publish_alert %>"
```

Comment:
Not sure how much difference it makes but this way seems slightly more readable: https://gist.github.com/Rosa-Fox/d1482f3137599a1c8e4603da88d719a0